packages/aws-cdk-lib/aws-cognito/test/user-pool-idps/base.test.ts

```diff
@@ -39,13 +39,15 @@ describe('UserPoolIdentityProvider', () => {
         attributeMapping: {
           givenName: ProviderAttribute.FACEBOOK_NAME,
           birthdate: ProviderAttribute.FACEBOOK_BIRTHDAY,
+          emailVerified: ProviderAttribute.other('email_verified'),
```

Comment:
This is not a good user experience as users would need to manually figure out the exact text for email verified. I would recommend explicitly building something like 
```
public static readonly EMAIL_VERIFIED = new ProviderAttribute('email_verified');
```

Reply:
@GavinZZ 
Added constants for when idp is either apple or google.